package.json

```diff
@@ -18,7 +18,6 @@
     "http-server": "^0.12.3",
     "import-webflow-export": "^0.0.4",
     "npm-run-all": "^4.1.5",
-    "parcel": "^1.12.4",
```

Comment:
not sure how this ended up in here but, we do not need both right? Everything seems to work just fine with only `parcel-bundler`. This is also what the docs say https://parceljs.org/getting_started.html :)

Reply:
agreed